### PR TITLE
Improve build - test against real latest

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,10 +27,19 @@ jobs:
...
-              - "venv"
+      - run:
+          name: Install dependencies (dash)
+          command: |
+              git clone git@github.com:plotly/dash-renderer.git
+              git clone git@github.com:plotly/dash-core-components.git
+              git clone git@github.com:plotly/dash-html-components.git
+              git clone git@github.com:plotly/dash-table.git
+              . venv/bin/activate
+              pip install -e . --quiet
```
Comment (alexcjohnson) [inline .circleci/config.yml]: possibly more efficient to put this to after all the other `install -e` calls, so it doesn't try to install release versions of the rest of core only to override with the `-e` source versions later?

Comment (Marc-Andre-Rivet) [reply .circleci/config.yml]: will test installing it with `--no-deps`. The builds `generate_class` require dash to be in the environment.

Comment (Marc-Andre-Rivet) [reply .circleci/config.yml]: if it works, will apply to the other prs as they would have the same issue

Comment (Marc-Andre-Rivet) [reply .circleci/config.yml]: Causes this error https://circleci.com/gh/plotly/dash/3285 will building renderer

Obtaining file:///home/circleci/project/dash-renderer
dash 0.38.0 requires dash-core-components==0.43.1, which is not installed.
dash 0.38.0 requires dash-html-components==0.13.5, which is not installed.
dash 0.38.0 requires dash-table==3.5.0, which is not installed.
dash 0.38.0 requires Flask>=0.12, which is not installed.
dash 0.38.0 requires flask-compress, which is not installed.

**Pull request merged**